### PR TITLE
Mobx fix ar

### PR DESCRIPTION
**Pull request by soyarsauce**

### CODE REVIEW

**Comment by soyarsauce** [comment]
Restarted CI job

File: lib/ReactViews/Map/Navigation/AugmentedVirtualityTool.jsx [hunk 1]
```diff
@@ -36,6 +36,33 @@ class AugmentedVirtualityTool extends React.Component {
   handleClickAVTool() {
     // Make the AugmentedVirtuality module avaliable elsewhere.
     this.props.terria.augmentedVirtuality = this.augmentedVirtuality;
+    // feature detect for new ios 13
+    // it seems you don't need to ask for both, but who knows, ios 14 / something
+    // could change again
+    if (
+      DeviceMotionEvent &&
```
Comment (na9da) [inline lib/ReactViews/Map/Navigation/AugmentedVirtualityTool.jsx]: This should probably be `window.DeviceMotionEvent`  My chrome version doesn't have DeviceMotionEvent and it errors out saying DeviceMotionEvent is undefined.

Same with `DeviceOrientationEvent` below.

**Comment by na9da** [comment]
- [x] Works in Firefox
- [x] Works in Chrome for Android
- [ ] Safari
  I couldn't see the AR buttons on Safari (iOS version 13.3.1). Same with Chrome on iOS (80.0.3) but works well in Firefox iOS.

image


There is also a problem with the alignment of buttons in Chrome which is not new. I have logged it here: https://github.com/TerriaJS/terriajs/issues/4090

File: lib/ReactViews/Map/Navigation/AugmentedVirtualityTool.jsx (hunk 1 above)
Comment (soyarsauce) [reply lib/ReactViews/Map/Navigation/AugmentedVirtualityTool.jsx]: Thanks Nanda this is a good catch, I was too focused on whether `requestPermission` existed!

**Comment by na9da** [comment]
- [x] Confirmed that it works in Safari & Chrome for iOS in portrait mode.

**Comment by soyarsauce** [comment]
https://github.com/TerriaJS/terriajs/issues/4097

**Pull request merged**